src/filesplugin.js

```diff
@@ -263,100 +344,21 @@ OCA.Slack.SlackSendModalVue.$on('closed', () => {
-	if (filesToSend.length === 0) {
-		return
-	}
-
 	OCA.Slack.filesToSend = filesToSend
-
 	if (type === SEND_TYPE.public_link.id) {
-		sendPublicLinks(channelId, channelName, comment, permission, expirationDate, password).then(() => {
-			showSuccess(
-				n(
-					'integration_slack',
-					'A link to {fileName} was sent to {channelName}',
-					'All of the {number} links were sent to {channelName}',
-					OCA.Slack.filesToSend.length,
-					{
-						fileName: OCA.Slack.filesToSend[0].name,
-						channelName,
-						number: OCA.Slack.filesToSend.length,
-					}
-				)
-			)
-			OCA.Slack.SlackSendModalVue.success()
-		}).catch((error) => {
-			errorCallback(error)
-			showError(
-				t('integration_slack', 'Failed to send links to Slack')
-				+ ' ' + error.response?.request?.responseText
-			)
-		})
+		sendPublicLinks(channelId, channelName, comment, permission, expirationDate, password)
 	} else if (type === SEND_TYPE.internal_link.id) {
-		sendInternalLinks(channelId, comment).then(() => {
-			showSuccess(
-				n(
-					'integration_slack',
-					'A link to {fileName} was sent to {channelName}',
-					'All of the {number} links were sent to {channelName}',
-					OCA.Slack.filesToSend.length,
-					{
-						fileName: OCA.Slack.filesToSend[0].name,
-						number: OCA.Slack.filesToSend.length,
-						channelName,
-					}
-				)
-			)
-			OCA.Slack.SlackSendModalVue.success()
-		}).catch((error) => {
-			errorCallback(error)
-			showError(
-				n(
-					'integration_slack',
-					'Failed to send the internal link to {channelName}',
-					'Failed to send internal links to {channelName}',
-					OCA.Slack.filesToSend.length,
-					{
-						fileName: OCA.Slack.filesToSend[0].name,
-						channelName,
-					}
-				)
-				+ ': ' + error.response?.request?.responseText
-			)
-		})
+		sendInternalLinks(channelId, channelName, comment)
 	} else {
 		OCA.Slack.remoteFileIds = []
 		OCA.Slack.sentFileNames = []
-		OCA.Slack.filesToSend = filesToSend.filter((f) => f.type !== 'dir')
-
-		Promise.all(OCA.Slack.filesToSend.map(sendFile(channelId, channelName, comment))).then(() => {
-			showSuccess(
-				n(
-					'integration_slack',
-					'{fileName} was successfully sent to {channelName}',
-					'All of the {number} files were sent to {channelName}',
-					OCA.Slack.filesToSend.length,
-					{
-						fileName: OCA.Slack.filesToSend[0].name,
-						number: OCA.Slack.filesToSend.length,
-						channelName,
-					}
-				)
-			)
-			OCA.Slack.SlackSendModalVue.success()
-		}).catch(errorCallback)
+		sendFileLoop(channelId, channelName, comment)
 	}
 })
 
-function errorCallback(error) {
-	console.error(error)
-	OCA.Slack.SlackSendModalVue.failure()
-	OCA.Slack.filesToSend = []
-	OCA.Slack.sentFileNames = []
-}
-
 // get Slack state
-axios.get(IS_CONNECTED_URL).then((response) => {
+const urlCheckConnection = generateUrl('/apps/integration_slac/is-connected')
```

Comment:
'/apps/integration_slack/is-connected'  
That one took long to pin point :)

Reply:
oops

Reply:
I checked the whole file and I only made this typo once.